src/pkg/caendr/caendr/models/datastore/database_operation.py

```diff
@@ -62,32 +100,22 @@ def get_props_set(cls):
     }
 
 
-
-  ## Special Properties ##
-
+  #
+  # Database Operation prop
+  #
 
   @property
   def db_operation(self):
-    return self.__db_operation
+    return self._get_enum_prop(DbOp, 'db_operation', None)
```

Comment:
Implemented & standardized `Enum` props using `Entity._set_enum_prop` and `Entity._get_enum_prop`